mkdocs.yml

```diff
@@ -72,6 +72,8 @@ nav:
     - Tutorials:
         - Connect to the JSON-RPC node proxy: Tutorials/JsonRPCProxy.md
         - Sign with EDDSA and BN254: Tutorials/EddsaBN254Sig.md
+        - Connect to Infura: Tutorials/ConnectInfura.md
```

Comment:
Should this be `Connect to Infura Endpoint` to avoid confusion?  This is how it was referred to by the links from the metaTXN page and the title of the page itself.